packages/application/src/router.ts

```diff
@@ -7,158 +7,13 @@ import { URLExt } from '@jupyterlab/coreutils';
 
 import { CommandRegistry } from '@phosphor/commands';
 
-import {
-  PromiseDelegate,
-  ReadonlyJSONObject,
-  Token
-} from '@phosphor/coreutils';
+import { PromiseDelegate, Token } from '@phosphor/coreutils';
 
 import { DisposableDelegate, IDisposable } from '@phosphor/disposable';
 
 import { ISignal, Signal } from '@phosphor/signaling';
 
-/* tslint:disable */
```

Comment:
I moved the router token and interfaces to `application/src/token.ts` as per #6334